Enable/Utilities/Utilities.m

```diff
@@ -279,36 +300,54 @@ + (void) postReviewWithLocation:(Location * _Nonnull) location rating: (int) rat
             completion(error);
             return;
         } else {
-            review.userProfileID = profile;
-            review.title = title;
-            review.reviewText = description;
-            review.rating = rating;
-            review.locationID = location;
-            review.images = (NSArray *)parseFiles;
-            review.likes = 0;
-        }
-
-        [review saveInBackgroundWithBlock:^(BOOL succeeded, NSError * _Nullable error) {
-            if(!error){
-                if(succeeded){
-                    NSLog(@"Successful post review");
-                    completion(nil);
-                } else {
-                    NSLog(@"Fail saveReviewInBackground (in Post Review) couldn't save.");
-                    NSDictionary * errorDict = @{@"title" : @"Failed to post review",
-                                                 @"message" : @"Couldn't save review in background",
-                                                 @"code" : [NSNumber numberWithInt: kCustomizedErrorCode]};
+            
+            [location incrementKey:@"reviews" byAmount:[NSNumber numberWithInt:1]];
+            [location setRating: (location.rating * (location.reviews - 1) + rating) / location.reviews];
```

Comment:
make a helper function to compute the average, it'll make the code easier to read.

Note: computing the average rating on the front end is fine for this project, but you should do that on a backend otherwise